llvm-spirv/tools/llvm-spirv/llvm-spirv.cpp

```diff
@@ -154,15 +154,8 @@ static cl::opt<bool>
     SPIRVToolsDis("spirv-tools-dis", cl::init(false),
                   cl::desc("Emit textual assembly using SPIRV-Tools"));
 
-#if SPIRV_ENABLE_OPAQUE_POINTERS
-constexpr static bool SPIRVOpaquePointersDefault = true;
-#else
-constexpr static bool SPIRVOpaquePointersDefault = false;
-#endif
-
 static cl::opt<bool>
-    EmitOpaquePointers("emit-opaque-pointers",
-                       cl::init(SPIRVOpaquePointersDefault),
+    EmitOpaquePointers("emit-opaque-pointers", cl::init(true),
```

Comment:
I think we still need to keep this change until we are ready to switch the whole stack to opaque pointers.
To test the status, it's better to change the CMAKE variable which will turn on opaque pointers for all components.
Here as an example how I tested it - https://github.com/intel/llvm/pull/9828/files#diff-f6c3fac0de0e2bbd79ac93b0a6aabe77d9961c5487347a2170c78ac77770d8afR895.

Reply:
Restored

Reply:
Great, but I'd like to understand the status of SYCL tests for DPC++ compiler using opaque pointers.
We need to test this in addition to regular CI pre-commit. I'd like to make sure that check-llvm-spirv test suite passes both settings w/ and w/o opaque pointers.

Reply:
@MrSidims, could you test this configuration, please?